samples/Config.CustomConfigurationProviders.Sample/Config.CustomConfigurationProviders.Sample.xproj

```diff
@@ -9,7 +9,7 @@
     <ProjectGuid>c2c08dd5-22ea-48dd-bda4-588dcd38048f</ProjectGuid>
     <RootNamespace>Microsoft.Extensions.Configuration.ConfigFile</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\</OutputPath>
```

Comment:
This was changed by the latest (the last recommended) tooling. I will revert it.